fetch/nosniff/resources/nosniff-weird-tokens.asis

```diff
@@ -0,0 +1,6 @@
+HTTP/1.1 200 YOU HAVE NO POWER HERE
+Content-Length: 22
+Content-Type: x/x
+X-Content-Type-options: nosniff,,@#$#%%&^&^*()()11!
```

Comment:
Could also do a weird tokens test with the "@#$#%%&^&^*()()11!" first, to make sure their headers with weird tokens aren't just ignored.  Could additionally / alternatively do something similar for the tabulation test as well, adding another nosniff after the comma.